charts/vela-core/templates/defwithtemplate/update-strategy.yaml

```diff
@@ -0,0 +1,77 @@
+# Code generated by KubeVela templates. DO NOT EDIT. Please edit the original cue file.
+# Definition source cue file: vela-templates/definitions/internal/update-strategy.cue
+apiVersion: core.oam.dev/v1beta1
+kind: TraitDefinition
+metadata:
+  annotations:
+    definition.oam.dev/alias: ""
+    definition.oam.dev/description: Set update strategy for Deployment/DaemonSet/StatefulSet
+  name: update-strategy
```

Comment:
```suggestion
  name: native-rollout
```

Reply:
`update-strategy` is a bit confusion while `native-rollout` is also not a good enough name, any other ideas? @StevenLeiZhang

Reply:
> `update-strategy` is a bit confusion while `native-rollout` is also not a good enough name, any other ideas? @StevenLeiZhang

what about k8s-update-strategy? It is similar to k8s-object and kruise-rollout, which indicates Provider and Intent/Target?

Reply:
@StevenLeiZhang OK

Reply:
> @StevenLeiZhang OK

updated